src/lib/helium_us915_netbeginregioninit.cpp

```diff
@@ -46,7 +46,7 @@ void Arduino_LoRaWAN_Helium_us915::NetBeginRegionInit()
     // Helium is an 8-channel network on subband 6 of [0..7]. Pre-join
     // we therefore want to limit to probing channels 48~55 / 70.
```

Comment:
Please update the comment  -- subband 1 of [0..7]. Prejoin, we therefore want to limit to probing channels 8~15/65.

Reply:
@terrillmoore Changes have been done